### PR TITLE
Update to Dotnet core 5

### PR DESCRIPTION
**Pull request by XiaoxuHU**

### CODE REVIEW

File: ApplensBackend/AppLensV3.csproj [hunk 1]
```diff
@@ -18,22 +18,24 @@
...
     <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.9.2" />
     <PackageReference Include="Microsoft.Azure.Kusto.Cloud.Platform" Version="10.0.1" />
     <PackageReference Include="Microsoft.Azure.Kusto.Data" Version="10.0.1" />
+	<PackageReference Include="Microsoft.Extensions.Logging.AzureAppServices" Version="3.1.9" />
     <PackageReference Include="Microsoft.Identity.Client" Version="4.42.1" />
     <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="5.2.8" />
     <PackageReference Include="Octokit" Version="0.29.0" />
     <PackageReference Include="Sendgrid" Version="9.10.0" />
     <PackageReference Include="StyleCop.Analyzers" Version="1.1.1-rc.108" />
+    <PackageReference Include="WindowsAzure.Storage" Version="9.3.3" />
```
Comment (rekhaswaminathan) [inline ApplensBackend/AppLensV3.csproj:38]: do we use this package anywhere in the Portal code?

File: ApplensBackend/Services/Authorization/AuthorizationHandler.cs [hunk 2]
```diff
@@ -97,8 +107,8 @@ public SecurityGroupHandler(IHttpContextAccessor httpContextAccessor, IConfigura
             var applensTesters = new SecurityGroupConfig();
             configuration.Bind("ApplensAccess", applensAccess);
             configuration.Bind("ApplensTesters", applensTesters);
-            loggedInUsersCache.Add(applensAccess.GroupId, new Dictionary<string, CachedUser>());
-            loggedInUsersCache.Add(applensTesters.GroupId, new Dictionary<string, CachedUser>());
+            loggedInUsersCache[applensAccess.GroupId] = new Dictionary<string, CachedUser>();
+            loggedInUsersCache[applensTesters.GroupId] = new Dictionary<string, CachedUser>();
```
Comment (rekhaswaminathan) [inline ApplensBackend/Services/Authorization/AuthorizationHandler.cs:111]: Same statement is added twice?

**Comment by cindylovescoding** [comment]
Are we gonna migrate to 5.0 directly?

**Comment by XiaoxuHU** [comment]
> Are we gonna migrate to 5.0 directly?

Yes, 3.1 and 5.0 are almost same. No need code changes migrate from 3.1 to 5.0

File: ApplensBackend/Services/Authorization/AuthorizationHandler.cs (hunk 2 above)
Comment (rekhaswaminathan) [reply ApplensBackend/Services/Authorization/AuthorizationHandler.cs:111]: Ah got it. Missed it.

File: ApplensBackend/AppLensV3.csproj (hunk 1 above)
Comment (XiaoxuHU) [reply ApplensBackend/AppLensV3.csproj:38]: @rekhaswaminathan , this package is using in [LocalDevelopmentClientService.cs](https://github.com/Azure/Azure-AppServices-Diagnostics-Portal/blob/main/ApplensBackend/Services/LocalDevelopmentClientService/LocalDevelopmentClientService.cs#L148)

**Review by rekhaswaminathan — APPROVED** [review]

**Review by SharmaAjay19 — APPROVED** [review]
Looks good to me. Thanks for handling the SecurityGroupLocalDevelopment thing.

**Review by hforeste — APPROVED** [review]

**Pull request merged**